maistra/run-test-infra-script.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Could you move this file to the `maistra/scripts` directory ?

Reply:
Done!